ansible/roles/formplayer/vars/main.yml

```diff
@@ -1,7 +1,11 @@
 formplayer_build_dir: "{{ code_home }}/formplayer_build"
 formplayer_data_dir: "{{ encrypted_root }}/formplayer"
+# formplayer_sentry_dsn: ..defined in secret file
 # formplayer_db_name: ..defined in secret file
 hq_main_db_host: "{% for config in postgresql_dbs -%}
-  {%- if config.name == localsettings.PG_DATABASE_NAME -%}{{ config.host }}
+  {%- if config.name == localsettings.PG_DATABASE_NAME -%}
+  {%- if 'host' in config -%}{{ config.host }}
+  {% else %}{{ groups.postgresql.0 }}
+  {%- endif -%}
```

Comment:
Just learnt about Ansible's JSON Query filter (https://docs.ansible.com/ansible/playbooks_filters.html#json-query-filter):
```
hq_main_db_host: {{ postgresql_dbs|json_query('[*][?django_alias==\'default\'] | [0].host')|default(groups.postgresql.0) }}
```
Note: not tested - also fine to leave this as is - just thought I'd share